### PR TITLE
add login

### PR DESCRIPTION
**Pull request by leewaichong**

### CODE REVIEW

File: .env
Comment (Aung-Phone-Naing) [inline .env:1]: Remove this file.

File: backend/src/types/express/index.d.ts
Comment (Aung-Phone-Naing) [inline backend/src/types/express/index.d.ts:1]: Whats this declaration for?

**Review by Aung-Phone-Naing — CHANGES_REQUESTED** [review]
I think generally all goods, just remove the .env and answer the comment for the declaration. Perhaps you can take a look at the structure for the schema branch, I have firmed up the routes, handlers and query functions for a cleaner approach.

File: backend/src/types/express/index.d.ts
Comment (leewaichong) [reply backend/src/types/express/index.d.ts:1]: It helps with TypeScript typing so that `req.user` is a valid property, and it can provide type checking.

**Pull request merged**